style.css

```diff
@@ -0,0 +1,568 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Poppins", sans-serif;
+  background: #f4f5f7;
+}
+
+@media (min-width: 992px) and (max-width: 1900px) {
+  nav {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    background-color: #fff;
+    height: 118px;
+  }
+
+  nav a {
+    font-weight: bold;
+    font-size: 18px;
+    line-height: 20px;
+    color: #6070ff;
+    text-decoration: none;
+    width: 76px;
+    height: 20px;
+    margin: 28px 143px;
+    padding: 10px 12px;
+  }
+
+  #list {
+    margin: 15px 142px;
+    padding: 0;
+  }
+
+  #list-2 {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    list-style-type: none;
+  }
+
+  #portfolio {
+    padding: 10px 12px;
+    color: #344563;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  #about {
+    padding: 10px 12px;
+    color: #344563;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  #contact {
+    padding: 10px 12px;
+    color: #344563;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  header {
+    padding: 0 8px 76.54px 8px;
+    background: url("./images/Ellipse3.png") top right no-repeat, url("./images/Ellipse4.png") top left no-repeat, url("./images/Ellipse5.png") top no-repeat, url("./images/Ellipse8.png") top right no-repeat, url("./images/Ellipse11.png") left top no-repeat, url("./images/Ellipse12.png") 341.76px 584.5px no-repeat, url("./images/Ellipse13.png") 332.18px 596.46px no-repeat, url("./images/Ellipse14.png") 298px 271px no-repeat, url("./images/Ellipse15.png") 53px 550px no-repeat, url("./images/Rectangle4.png") top right no-repeat, url("./images/Rectangle5.png") 191.82px 560px no-repeat, url("./images/Rectangle6.png") -25px 77px no-repeat, url("./images/Rectangle7.png") 127.18px 99px no-repeat;
+    background-color: #fff;
+    border-radius: 0 0 0 70px;
+  }
+
+  header h1 {
+    font-weight: bold;
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+    padding: 241px 260px 6px 260px;
+    color: #091e42;
+    background-color: #fff;
+    margin: 0;
```

Comment:
- [ ] Hello, You've implemented the background of the main section really well. Although it can't be seen because the styling on the `h1` has a background color of `#fff` that is blocking the background image. please remove this `background-color`.

Your Design | Reference Design
-|-
![image](https://user-images.githubusercontent.com/18730106/155593136-70945763-5b1e-4c15-918a-9b584c93608f.png)|![image](https://user-images.githubusercontent.com/18730106/155593152-34090d32-8dca-426b-aac4-7840eefa61ad.png)